data/geany.glade

```diff
@@ -6321,6 +6321,7 @@
               <object class="GtkMenuBar" id="menubar1">
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
+                <signal name="deactivate" handler="on_menubar1_deactivate" after="yes" swapped="no"/>
```

Comment:
Sad we can't really connect something on the menu item next to the "activate" signal.  We could use "deselect", but I'm not sure is really correct.